packages/ui-components/src/components/labels-dropdown-input-item/labels-dropdown-input-item.tsx

```diff
@@ -0,0 +1,33 @@
+import { Component, Event, EventEmitter, Prop, h } from '@stencil/core';
+import { LabelsInputItem, LabelsInputItemEvents } from './labels-dropdown-input-item.types';
+import { EIconName } from '../icon/icon.types';
+
+@Component({
+	tag: 'kv-labels-dropdown-input-item',
+	styleUrl: 'labels-dropdown-input-item.scss',
+	shadow: true
+})
+export class KvLabelsDropdownInputItem implements LabelsInputItem, LabelsInputItemEvents {
+	/** @inheritdoc */
+	@Prop({ reflect: true }) value!: string;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) label!: string;
+
+	/** @inheritdoc */
+	@Event() removeClick: EventEmitter<string>;
+
+	render() {
+		return (
+			<div class="labels-input-item" onClick={e => e.stopPropagation()}>
```

Comment:
Why do we need the `stopPropagation` here?

Reply:
The `kv-dropdown` component captures click events on the text-field through the `onClick` event, clicks on these labels were bubbling up and causing the dropdown to open when there was a click on one of these items, hence the need to stop event propagation, this seemed like the simplest way of handling this case but let me know if you have any other suggestions ;)